File: exs/mundo_1/ruby/030.rb
=begin
Desafio 030

Problema:  Crie um programa que leia um número inteiro e mostre na tela se ele é PAR ou ÍMPAR.

Resolução do problema:
=end

print'Digite um número: '
n = gets.chomp.to_i

if n % 2 == 0
	puts"O número #{n} é par."
else
	puts"O número #{n} é ímpar."
end

File: exs/mundo_1/golang/030.go
/*
Desafio 030

Problema:  Crie um programa que leia um número inteiro e mostre na tela se ele é PAR ou ÍMPAR.

Resolução do problema:
*/

package main

import (
	"fmt"
)

var n int 

func main(){
	fmt.Print("Digite um número inteiro: ")
	fmt.Scanln(&n)

	if n % 2 == 0{
		fmt.Printf("O número %v é par.\n", n)
	} else {
		fmt.Printf("O número %v é ímpar.\n", n)
	}
}
